src/toTransform.js

```diff
@@ -42,10 +42,76 @@
     return 'translate(' + valuesArray.join(', ') + ')';
   }
 
+  function convertRotate (values) {
+    if (values === undefined || values === 'none') {
+      return null;
+    }
+
+    var valuesArray = values.split(' ');
+    var numValues = valuesArray.length;
+
+    if (numValues !== 4 && numValues !== 1) {
+      throw new InvalidArgument('Incorrect number of arguments for rotate');
+    }
+
+    if (numValues === 1 && valuesArray[0] === '') {
+      throw new InvalidArgument('Empty string is not a valid argument');
+    }
+
+    var angleUnits = ['deg', 'grad', 'rad', 'turn'];
+    var angleUnit = valuesArray[numValues - 1];
+
+    var foundUnit = 0;
+    var unit = angleUnit.substring(angleUnit.length - 4, angleUnit.length);
+
+    if (angleUnits.indexOf(unit) !== -1) {
+      foundUnit = 1;
+    }
+
+    if (foundUnit === 0) {
+      unit = angleUnit.substring(angleUnit.length - 3, angleUnit.length);
+      if (angleUnits.indexOf(unit) !== -1) {
+        foundUnit = 1;
+      }
+    }
```

Comment:
We should avoid magic numbers like 3 and 4 and find a more elegant way to detect the unit.